### DISCUSSION TITLE
DNS over HTTPS

### DISCUSSION THREAD

**Discussion by jsalaz-Gramps** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#682] [Q&A]
My DNS entries are unencrypted (Windows 11) using DNS.  Is there a DNS over HTTPS template I can use?

**Comment by spirillen** [comment]
You should ask your question here: https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/issues/590

However, it will be an ever struggle between the good and bad guys, so my personal advise to you, as windows user, is to run a locally hosted [UNBOUND](https://nlnetlabs.nl/projects/unbound/download/) recursor, whish is the only real DNS recursor that support RPZ ~~(partially)~~ for windows. You can then setup encrypted dns recursion.

UPDATE: grammar and spelling

**Reply by spirillen** [reply]
Unbound support the full RPZ rule spectrum today.
